ci/npm-publish.ps1

```diff
@@ -34,8 +34,8 @@ try
 	}
 	else
 	{
-		Write-Host "Publishing..."
-		npm publish "$Tarball" "--access=$Access"
+		Write-Host "Publishing..." # workaround for https://github.com/actions/setup-node/issues/1440
+		NODE_AUTH_TOKEN="" npm publish "$Tarball" "--access=$Access"
```

Comment:
It’s not clear that the comment refers to the env variable on the line below

Reply:
Bon point, J'ai poussé un fix